src/nvim/ex_cmds_defs.h

```diff
@@ -176,6 +176,7 @@ typedef struct {
   int lockmarks;                        /* TRUE when ":lockmarks" was used */
   int keeppatterns;                     /* TRUE when ":keeppatterns" was used */
```

Comment:
Neovim uses C99 so it has a bool type.  Vim only requires C89, so int is used even for "boolean" values.  Types are gradually being changed as needed.  Since `browse` was being re-added, I figured it made sense to make it a bool.